types/webpack-dev-server/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
+    "http-proxy-middleware": "^1.0.0"
```

Comment:
IMO this cannot be done without package upgrade itself, the version 3.1 does not support that version of
`http-proxy-middleware`:
https://github.com/webpack/webpack-dev-server/blob/v3.11.2/package.json#L48